src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -38,7 +38,7 @@ <h3 class="sidebar-header">My workflows</h3>
         <div class="clearfix sidebar-header">
           <h3 class="sidebar-header">Shared with me</h3>
         </div>
-        <p *ngIf="groupSharedEntriesObject == undefined || groupEntriesObject == null || groupSharedEntriesObject.length == 0">You have no workflows shared with you</p>
+        <p *ngIf="groupSharedEntriesObject == undefined || groupSharedEntriesObject == null || groupSharedEntriesObject.length == 0">You have no workflows shared with you</p>
```

Comment:
I would just do `!groupSharedEntriesObject || !groupSharedEntriesObject.length`, but that might be a bad habit from pre-Typescript days. :) But yours works too.

Reply:
Ya whenever I type those checks I always feel like there must be a better way. That seems a lot better.